sw/device/tests/crypto/cryptotest/firmware/BUILD

```diff
@@ -161,3 +162,32 @@ opentitan_binary(
         "//sw/device/tests/crypto/cryptotest/json:ibex_fi_commands",
     ],
 )
+
+opentitan_test(
+    name = "chip_pen_test",
+    srcs = [":firmware.c"],
+    exec_env = {
+        "//hw/top_earlgrey:silicon_owner_sival_rom_ext": "silicon_owner",
+    },
+    silicon_owner = silicon_params(
+        tags = ["broken"],
+    ),
+    deps = [
+        ":aes",
+        ":aes_sca",
```

Comment:
We're missing imports for the new dependencies in master: `:hash` and `//sw/device/tests/crypto/cryptotest/json:hash_commands`.

Reply:
Thanks for your comment, I've created a new PR #21605 as I cannot rebase this PR myself.